build_helpers/cssTransformLoader.js

```diff
@@ -14,11 +14,6 @@ function slashTransform(content) {
 }
 
 module.exports = function(content) {
-  if (this && this.cacheable) {
```

Comment:
Any idea if these results are still cached in webpack3?  Or how this API changed?